docs/device-agent/quickstart.md

```diff
@@ -9,29 +9,29 @@ navOrder: 2
 
 ### Install Device Agent Software
 
-1. Open your Command Prompt (CMD) as an administrator.
+1. Open a Terminal/Command Prompt as an root/administrative user
 2. Run the following commands in sequence:
 
 Windows:
 ```bash
 mkdir c:\opt\flowfuse-device
 cd c:\opt\flowfuse-device
 npm install -g @flowfuse/device-agent
-flowfuse-device-agent -w --ui-user admin --ui-pass password --ui-port 8081
+flowfuse-device-agent -ui --ui-user admin --ui-pass password --ui-port 8081
```

Comment:
Should this be `--ui` rather than `-ui` ? If so, there are a couple other places that needs fixing below.

Reply:
Well spotted 🦅👁️